docs/guides.md

```diff
@@ -544,67 +544,176 @@ secureboot-certs
 ```
 
 :::tip
-Running `secureboot-certs` only needs to be done once per host, not per VM.  It must also be called after ISO upgrades once per upgraded host.
+Running `secureboot-certs` only needs to be done once per host, not per VM. It is not pool-wide. It must also be called after ISO upgrades once per upgraded host.
```

Comment:
"It is not pool-wide": I could be asking myself "what do they mean? It means you *need* to do it to every host in the pool or that you don't need to do it to every host since it's not poolwide?"

Reply:
Cool, I'll rephrase as "It must be called for every host in the pool."

Reply:
... "used to boot Secure Boot enabled VMs".

Reply:
"It must be called for every host in the pool." is enough because it is a rule that all pool members must be alike.